style.css

```diff
@@ -578,9 +577,148 @@
   width: 100%;
 }
 
+/* mobile popup window */
+
+.popup-container {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  max-height: 90vh;
+  padding: 25px 15px 25px 15px;
+  gap: 30px;
+  background-color: #fff;
+  position: fixed;
+  top: 25px;
+  left: 12px;
+  right: 12px;
+  z-index: 6;
+  overflow-y: scroll;
+}
+
+.popup-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.project-technologies-1 {
+  height: 50%;
+  gap: 10px;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+}
+
+.project-details-1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.more-project-details {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}

```

Comment:
- [ ] Good job so far :clap: , However, Please ensure that you your popup design matches the given design as a requirement, and please do not forget to reduce the size of your popup as well as its taking up the whole screen. matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student Popup](https://github.com/FilbertKimasa/Portfolio/assets/43528594/cf18329d-54ef-4bba-9981-d019975bc4c6)

</td>
      <td>

![Design Popup](https://github.com/FilbertKimasa/Portfolio/assets/43528594/7cffdcca-6df2-4c79-b499-b66266469613)

</td>
    </tr>
  </table>

Reply:
Hello @Alusp  👋... 
Thanks for the review, but I have used the breakpoint of 1350px in my media query because 768px was enough for only two cards in a row. should I change the popup window break point to 768px?
Starting from 1350px the popup design match with that in the figma except for the image.

Reply:
Hello @FilbertKimasa, you could try it out to see if it matches the given design, the most important thing is to make sure your buttons are aligned to the right and not to the left, and also consider resizing the height of the popup card to fit the screen. I hope this answers your question. :slightly_smiling_face: